natgeo/mortar
# AEM-6289 Render external images in Most Read component

https://jira.natgeo.com/browse/AEM-6289

This change updates the most read component to accept external images in the same manner the story card's image component does

To test:

- Pull AEM-6289
- Run `npm test` to verify tests
- Run `gulp serve`
- Verify Chris P. Bacon image is shown in the first slot of the most read component here http://localhost:3000/contentPackages/
- Verify 400x300 placehold.it image still renders in top spot of Most Read here http://localhost:3000/contentPackages/contentPackage/index.html
